game_patch/object/entity.cpp

```diff
@@ -201,6 +202,79 @@ CodeInjection entity_process_pre_hide_riot_shield_injection{
     },
 };
 
+FunHook<void(int)> entity_blood_throw_gibs_hook{
+    0x0042E3C0, [](int handle) {
+        if (g_game_config.gibbing && !rf::is_multi) {
```

Comment:
I would prefer to just return here if conditions are not fulfilled and avoid indenting all the code.
Also I feel like checking `g_game_config.gibbing` and `rf::is_multi` in 3 places is redundant. Maybe we could get away with checking it in one place?

Reply:
> Maybe we could get away with checking it in one place?

My understanding based on the code is that no, it needs to be checked in all 3 places to avoid issues. I added the checks in each case specifically because I found issues.

As noted above though, I have consolidated the common checks necessary everywhere into a bool.